spec/features/5_photos_spec.rb

```diff
@@ -1,120 +1,126 @@
+      @user_3 = FactoryGirl.create(:user, :username => "Three", :email => "three@m.com")
+      @photo_1 = FactoryGirl.create(:photo, :user_id => @user_1.id)
+      @photo_2 = FactoryGirl.create(:photo, :user_id => @user_2.id)
+      @photo_3 = FactoryGirl.create(:photo, :user_id => @user_3.id)
+    end
 
-feature "Photos:", js: do_not_show_tests_in_browser do
+    scenario "only show edit link for signed-in user's photos", points: 1 do
+      login_as(@user_2, :scope => :user)
+      visit "/photos"
 
-  scenario "only show edit/delete for signed-in user's photos", points: 2 do
-    user_1 = FactoryGirl.create(:user, :username => "1", :email => "1@m.com")
-    user_2 = FactoryGirl.create(:user, :username => "2", :email => "2@m.com")
-    photo_1 = FactoryGirl.create(:photo, :user_id => user_1.id)
-    photo_2 = FactoryGirl.create(:photo, :user_id => user_2.id)
-    login_as(user_2, :scope => :user)
+      @user_2.photos.each do |photo|
+        expect(page).to have_link(nil, href: "/photos/#{photo.id}/edit")
+      end
 
-    visit "/photos"
+      @user_1.photos.each do |photo|
+        expect(page).not_to have_link(nil, href: "/photos/#{photo.id}/edit")
+      end
+    end
 
-    expect(page).not_to have_link(nil, href: "/photos/#{photo_1.id}/edit")
-    expect(page).not_to have_link(nil, href: "/delete_photo/#{photo_1.id}")
-    expect(page).to have_link(nil, href: "/photos/#{photo_2.id}/edit")
-    expect(page).to have_link(nil, href: "/delete_photo/#{photo_2.id}")
-  end
+    scenario "only show delete link for signed-in user's photos", points: 1 do
+      login_as(@user_2, :scope => :user)
+      visit "/photos"
 
-  scenario "in /photos, Bootstrap panels used", points: 1 do
-    user = FactoryGirl.create(:user)
-    photo = FactoryGirl.create(:photo, :user_id => user.id)
-    login_as(user, :scope => :user)
+      @user_2.photos.each do |photo|
+        expect(page).to have_link(nil, href: "/delete_photo/#{photo.id}")
+      end
 
-    visit "/photos"
+      @user_1.photos.each do |photo|
+        expect(page).not_to have_link(nil, href: "/delete_photo/#{photo.id}")
+      end
+    end
 
-    expect(page).to have_css(".panel")
-  end
+    scenario "in /photos, Bootstrap panels used", points: 1 do
+      user = FactoryGirl.create(:user)
+      photo = FactoryGirl.create(:photo, :user_id => user.id)
+      login_as(user, :scope => :user)
 
-  scenario "in /photos, Font Awesome fa-plus icon used (and 'Photos' h1 tag isn't)", points: 1 do
-    user = FactoryGirl.create(:user)
-    photo = FactoryGirl.create(:photo, :user_id => user.id)
-    login_as(user, :scope => :user)
+      visit "/photos"
 
-    visit "/photos"
+      expect(page).to have_css(".panel")
+    end
 
-    expect(page).to have_css(".fa-plus")
-    expect(page).not_to have_selector("h1", text: "Photos")
-  end
+    scenario "in /photos, Font Awesome fa-plus icon used (and 'Photos' h1 tag isn't)", points: 1 do
```

Comment:
We should modify the target and the tests to not depend on Font Awesome, and instead use the standard text that is used by `draft:resource`s ("Show details", etc).

Reply:
(I've added access for you to the target on Heroku.)